src/discord/handle-voice-state-update.ts

```diff
@@ -27,7 +27,7 @@ export default function handleVoiceState(oldState: VoiceState, newState: VoiceSt
     return
   }
 
-  if (newChannelIsDiscordAudioChannel) {
+  if (newChannelIsDiscordAudioChannel && newChannelIsDiscordAudioChannel !== oldChannelIsDiscordAudioChannel) {
```

Comment:
What if we stored this in a variable for both if statements called `channelHasNotChanged`

Reply:
👍 i'll do that real quick